addons/auto-follow-topics/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Auto Follow Topics",
+  "description": "Automatically follow forum topics you post on",
```

Comment:
I think it should be "Automatically follow forum topics".

Reply:
Well yeah, the entire addon manifest requires review because I just scraped one together to be good enough for testing.

"Auto follow topics" sounds like the best to me, the forums tag is very clearly visible next to the addon name, so I don't think adding "forums" to the addon name is a good idea.